votor/src/consensus_pool/stats.rs

```diff
@@ -142,7 +142,7 @@ impl ConsensusPoolStats {
     fn report(&self) {
         datapoint_info!(
             "consensus_pool_stats",
-            ("conflicting_votes", self.conflicting_votes as i64, i64),
+            ("slashable_behavior", self.slashable_behavior as i64, i64),
```

Comment:
Why? conflicting_votes is self-explanatory, we might have slashable_behavior other than conflicting_votes, and I would prefer to list them separately in stats.

Reply:
Sorry, not sure I understand the feedback.

Please take a look at the rest of the PR.  `VotePool` is returning two types of errors: slashable behaviour and duplicate votes and the concept of conflicting votes is not there anymore.  This is tracking the slashable behaviour errors.  

Or are you saying that a validator may display slashable behaviour in ways other than voting?  And this metric will confuse those other behaviours with the voting slashable behaviour?